src/index.html

```diff
@@ -1,13 +1,590 @@
+                    <div class="testimonial__review__top">
+                      <div class="testimonial__review__block testimonial__review__block--1"></div>
+                      <img
+                        src="images/testimonials/Rectangle 13 2.svg"
+                        alt="Michel Grover"
+                        class="testimonial__review__image"
+                      >
+                      <div class="testimonial__review__block testimonial__review__block--2"></div>
+                    </div>
+                  </div>
+                  <div class="testimonial__review__content">
+                    <div class="testimonial__review__content-icon">
+                      <a href="#" class="icon testimonial--icon"></a>
+                      <p class="testimonial__review__text">
+                        AIR raises the agency bar to stratospheric heights
+                        on both creative output and client service.
+                      </p>
+                    </div>
+                    <div class="testimonial__review__content__bottom">
+                      <p class="testimonial__review__name">Michel Grover</p>
+                      <p class="section__subtitle testimonial__review__subtitle">Hulu</p>
+                    </div>
+                  </div>
+                </article>
+              </div>
+            </div>
+          </div>
+        </section>
+
+        <section class="cta" id="cta">
+          <div class="container">
+            <div class="cta__content">
+              <h2 class="cta__title">Vision, Passion, Results</h2>
+              <p class="cta__description">
+                We are sure that first-rate job
+                is possible only if all three components are united.
+              </p>
+              <a href="#" class="cta__button">Apply</a>
+            </div>
+          </div>
+        </section>
+      </div>
+    </main>
+    <footer class="footer" id="contact-us">
+      <div class="container">
+        <div class="footer__content">
+          <div class="footer__forms">
+            <h3 class="footer__title section__title">Send us  a message</h3>
+            <form
+              action="submit"
+              onsubmit="this.reset(); return false"
+              class="footer__form"
+            >
+              <input
+                type="email"
+                name="email"
+                class="input footer__input"
+                placeholder="Your email"
+                required
+              >
+              <input
+                type="text"
+                name="name"
+                class="footer__input"
+                placeholder="Your name"
+                required
+              >
+              <textarea
+                name="textarea"
+                class="footer__input footer__input--textarea"
+                placeholder="Your message"
+                required
+              >
+              </textarea>
```

Comment:
```suggestion
              ></textarea>
```
i think it should fix that problem
<img width="407" alt="image" src="https://github.com/mate-academy/layout_dia/assets/70220667/2031e4a1-a634-4f99-82a8-69b847709610">